### ISSUE TITLE
Add tasks for Apache config

### ISSUE THREAD

**Issue by afred** [WGBH/mla-playbooks#55]
**Done when** task(s) do the following:
- add sites-available/openvault.conf from a template (see comment below)
- symlink sites-enabled/openvault.conf
- add `include /etc/httpd/sites-enabled/` to /etc/httpd/conf/httpd.conf
- restart apache

**Comment by afred** [comment]
Virtual host config for the website should look something like this...

``` apache
<VirtualHost *:80>
  ServerName openvault

  DocumentRoot /var/www/openvault/current/public/
  RackBaseURI /
  RackEnv production

  <Directory /var/www/openvault/current/public/>
    Options FollowSymLinks -MultiViews 
    AllowOverride None
    Order allow,deny
    Allow from all
  </Directory>

  LogLevel info
  ErrorLog /var/log/httpd/openvault-error.log
  CustomLog /var/log/httpd/openvault-access.log combined
</VirtualHost>
```

**Comment by afred** [comment]
closed by #56
